opensuse-tomcat-image/pom.xml

```diff
@@ -122,6 +122,8 @@
                                     <http_proxy>${env.HTTP_PROXY}</http_proxy>
                                     <https_proxy>${env.HTTPS_PROXY}</https_proxy>
                                     <no_proxy>${env.NO_PROXY}</no_proxy>
+                                    <!-- Pass in opensuse-tomcat-jul image as param -->
+                                    <JUL_IMAGE>${dockerCafImagePrefix}opensuse-tomcat-jul${dockerProjectVersion}</JUL_IMAGE>
```

Comment:
<!-- Pass in opensuse-tomcat-juli image as param -->
                                    <JULI_IMAGE>${dockerCafImagePrefix}opensuse-tomcat-juli${dockerProjectVersion}</JULI_IMAGE>